src/sql/cmd/snippets.py

```diff
@@ -62,6 +63,26 @@ def snippets(others):
         help="Force delete all stored snippets",
         required=False,
     )
+    if len(others) == 1:
+        all_snippets = util.get_all_keys()
+        if others[0] in all_snippets:
+            return str(store[others[0]])
+
+        base_err_msg = f"'{others[0]}' is not a snippet. "
+        if len(all_snippets) == 0:
+            err_msg = "%sThere is no available snippet."
+        elif len(all_snippets) < 3:
+            err_msg = "%sAvailable snippets are " f"{' and '.join(all_snippets)}."
+        else:
+            err_msg = (
+                "%sAvailable snippets are "
+                f"{', '.join(all_snippets[:-1])}, and "
+                f"{all_snippets[-1]}."
+            )
+        err_msg = err_msg % (base_err_msg)
```

Comment:
I don't think we need to separate it for < 3 and > 3 just for printing the snippets. 
We can use `pretty_print` to print `all_snippets`

Reply:
To simplify it, I changed it to `', '.join(all_snippets)` if `all_snippets` is not empty which removed the if statement for < 3 and > 3